src/redux/products/products.actions.js

```diff
@@ -0,0 +1,17 @@
+import {
+  SET_ALL_PRODUCTS,
+  SET_LOADING,
+  GET_ALL_PRODUCTS
+} from './products.types';
+
+export const setAllProducts = (payload) => ({
+  type: SET_ALL_PRODUCTS,
+  payload
+});
+export const getAllProducts = () => ({
+  type: GET_ALL_PRODUCTS
+});
+export const setLoading = (loading) => ({
+  type: SET_LOADING,
```

Comment:
Please, rename it to `SET_PRODUCTS_LOADING` to distinguish from other loadings actions

Reply:
fixed